tests/test_adjoint_jacobian.py

```diff
@@ -22,7 +22,13 @@
 from pennylane import QNode, qnode
 from scipy.stats import unitary_group
 import pennylane_lightning_kokkos as plk
-from pennylane_lightning_kokkos.lightning_kokkos_qubit_ops import InitArguments
+from pennylane_lightning_kokkos.lightning_kokkos_qubit_ops import (
+    InitArguments,
+    NamedObsKokkos_C64,
```

Comment:
Can these be C128 (as in std::complex) as well, since that is the default we expect throughout.